streaming-binlog/pom.xml

```diff
@@ -139,6 +139,29 @@
 
   <build>
     <plugins>
+      <!-- Generates self-contained jars with dependencies-->
+      <plugin>
+        <artifactId>maven-assembly-plugin</artifactId>
```

Comment:
Have you looked into how we create the uber-jar for the batch mode? I mean is there a reason not to use the same plugin here and there (i.e., shade vs assembly).

Reply:
Thanks @bashir2, For some reasone I thought batch mode was using `maven-assembly-plugin` updating to `shade`

Reply:
I have switched this to shade, thanks!